manager/integration/tests/test_recurring_job.py

```diff
@@ -1239,66 +1224,107 @@ def test_recurring_job_delete(client, batch_v1_beta_api, volume_name):  # NOQA
     wait_for_cron_job_delete(batch_v1_beta_api, JOB_LABEL+"="+back2)
     wait_for_cron_job_delete(batch_v1_beta_api, JOB_LABEL+"="+back3)
 
-    # Should be able to delete recurring job while existing in volume label
+
+@pytest.mark.recurring_job  # NOQA
+def test_recurring_job_delete_should_remove_volume_label(client, batch_v1_beta_api, volume_name):  # NOQA
+    """
+    Scenario: test delete recurring job should remove volume labels
+
+    Given 1 volume created.
+    And  create `snapshot1` recurring job.
+         create `backup1`   recurring job.
+    And  `snapshot1` job applied to the volume.
+         `backup1`   job applied to the volume.
+    And  `snapshot1` job exist in volume recurring job label.
+         `backup1`   job exist in volume recurring job label.
+
+    When delete `snapshot1` recurring job.
+    Then `snapshot1` job not exist in volume recurring job label.
+         `backup1`   job     exist in volume recurring job label.
+
+    When delete `backup1` recurring job.
+    Then `snapshot1` job not exist in volume recurring job label.
+         `backup1`   job not exist in volume recurring job label.
+
+    Given create `snapshot1` recurring job with `group-1` in groups.
+    And   create `snapshot2` recurring job with `group-1` in groups.
+    And   create `backup1`   recurring job with `group-1` in groups.
+    And   create `backup2`   recurring job with `default` in groups.
+    // The default job-group automatically applies to the volumes
+    // with no recurring job. We want to keep the test focused on the
+    // behavior so removing the default job-group assignment first.
+    And   remove volume recurring job label `default` job-group.
+    And  `group-1` job-group     exist in volume recurring job label.
+         `default` job-group not exist in volume recurring job label.
```

Comment:
Will group `default` be re-added to the volume automatically? If YES, you should not check this.

Reply:
OK. Then please add this explanation here. And maybe we need to mention users as well when they try to remove group `default` from the volume. I already forgot this behavior...

Reply:
This is mentioned in [doc](https://longhorn.io/docs/1.2.3/snapshots-and-backups/scheduling-backups-and-snapshots/#set-up-recurring-jobs-to-be-the-default).